packages/app/background-jobs-common/src/background-job-processor/processors/AbstractBackgroundJobProcessor.ts

```diff
@@ -264,8 +264,10 @@ export abstract class AbstractBackgroundJobProcessor<
     const jobs =
       (await this._queue?.addBulk(
         jobData.map((data) => ({
-          name: this.config.queueId,
-          data,
+          // biome-ignore lint/suspicious/noExplicitAny: bull expects ExtractNameType<JobPayload>, but ExtractNameType is not exposed
+          name: this.config.queueId as any,
```

Comment:
I hope there is a better solution for this, I would be grateful if someone could look deeper into how we can make TS happy here

Reply:
I will try to have a look later 🙏 But if you can't find a better solution I doubt I will be able to do it 😅

Reply:
You are much better with TypeScript type wizardry than I am, so there is definitely hope :D